src/App.tsx

```diff
@@ -1,61 +1,159 @@
 import './App.scss';
```

Comment:
Consider renaming 'App.scss' to a more descriptive name like 'AppStyles.scss' to improve code readability.